src/callbacks/liquidity/BaseDTL.sol

```diff
@@ -34,24 +40,27 @@ abstract contract BaseDirectToLiquidity is BaseCallback {
 
     error Callback_LinearVestingModuleNotFound();
 
+    /// @notice The auction lot has already been completed
+    error Callback_AlreadyComplete();
+
     // ========== STRUCTS ========== //
 
     /// @notice     Configuration for the DTL callback
     ///
-    /// @param      recipient                   The recipient of the LP tokens
-    /// @param      lotCapacity                 The capacity of the lot
-    /// @param      lotCuratorPayout            The maximum curator payout of the lot
-    /// @param      proceedsUtilisationPercent  The percentage of the proceeds to deposit into the pool, in basis points (1% = 100)
-    /// @param      vestingStart                The start of the vesting period for the LP tokens (0 if disabled)
-    /// @param      vestingExpiry               The end of the vesting period for the LP tokens (0 if disabled)
-    /// @param      linearVestingModule         The LinearVesting module for the LP tokens (only set if linear vesting is enabled)
-    /// @param      active                      Whether the lot is active
-    /// @param      implParams                  The implementation-specific parameters
+    /// @param      recipient           Recipient of the LP tokens
+    /// @param      lotCapacity         Capacity of the lot
+    /// @param      lotCuratorPayout    Maximum curator payout of the lot
+    /// @param      poolPercent         Percentage of the proceeds to allocate to the pool, in basis points (1% = 100). The remainder will be sent to the `recipient`.
```

Comment:
> The remainder will be sent to the `recipient`.

This is inaccurate. It's sent to the seller currently.

In the Baseline DTL, it's sent to the specified recipient. I suggest we amend this to the recipient, in order to be consistent.

Reply:
Agree